test/ng/rootScopeSpec.js

```diff
@@ -1097,6 +1097,94 @@ describe('Scope', function() {
       expect(log).toEqual('');
     });
 
+    it('should remove all watchers once one-time/constant bindings are stable', function() {
```

Comment:
This seems unrelated, but there are no other watcher-count tests so I thought I'd leave it in...